chia/cmds/wallet.py

```diff
@@ -360,3 +360,29 @@ def cancel_offer_cmd(wallet_rpc_port: Optional[int], fingerprint: int, id: str,
     from .wallet_funcs import execute_with_wallet, cancel_offer
 
     asyncio.run(execute_with_wallet(wallet_rpc_port, fingerprint, extra_params, cancel_offer))
+
+
+@wallet_cmd.group("did", short_help="DID related actions")
+def did_cmd():
+    pass
+
+
+@did_cmd.command("create", short_help="Create DID wallet")
+@click.option(
+    "-wp",
+    "--wallet-rpc-port",
+    help="Set the port where the Wallet is hosting the RPC interface. See the rpc_port under wallet in config.yaml",
+    type=int,
+    default=None,
+)
+@click.option("-f", "--fingerprint", help="Set the fingerprint to specify which wallet to use", type=int)
```

Comment:
We'll need to add an option to set a fee. See `chia plotnft create --help` for an example (using `-m/--fee`). I'm not sure if creating a DID will require two spends as it does for plotnft creation. Feel free to revisit the fee issue in a separate PR.

@ytx1991 has `create_did_wallet()` been updated to take a fee parameter, and if so, is that fee applied twice as it is for the plotnft case?

Reply:
It wasn't but I changed it after saw this.